.github/workflows/operator-developer-e2e.yaml

```diff
@@ -0,0 +1,23 @@
+name: operator-developer-e2e
+
+on:
+  workflow_dispatch:
+  pull_request:
+  push:
+   branches:
+   - main
```

Comment:
Inconsistent indentation: should be 2 spaces (i.e. more than 1) to match the rest of the file.